platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/location/LocationComponent.java

```diff
@@ -835,16 +853,16 @@ public void onStartLoadingMap() {
    * Internal use.
    */
   public void onFinishLoadingStyle() {
```

Comment:
We should reset the style object here and push the new style to `LocationLayerController`.